### ISSUE TITLE
add option to load remembered timeline position by default when opening aria

### ISSUE THREAD

**Issue by foss-** [poppingmoon/aria#306]
### Summary

Add option to load remembered timeline position by default when opening aria.

### Purpose

Some users like reading from old to new. On app open aria allows them to jump to the timeline spot where they left last time. That is very useful, but if a user likes to read from old to new, it would be helpful to load that exact timeline position when aria is opened.

Maybe a setting `Remember timeline position` could cover this functionality.

### Do you want to implement this feature yourself?

- [ ] Yes, I will implement this by myself and send a pull request

**Comment by foss-** [comment]
Unsure what to make of the eyes emoji. Does the feature request not make sense for you? I recall there being some debate about this when rhe feature was worked on in tusky client. Two clients imo got it right and those are Pachli and Fedilab.

**Comment by poppingmoon** [comment]
I reacted with 👀 just because I have read it, no implication.

I think it is a good feature. I will add this after Misskey 2024.7.0 is released because it contains a fix about `sinceId` on fanout timelines (`https://github.com/misskey-dev/misskey/pull/14061`).

**Comment by foss-** [comment]
Misskey 2024.07 is out 🎉 https://github.com/misskey-dev/misskey/releases/tag/2024.7.0

I am testing 0.13.7 which appeared on F-Droid yesterday and includes https://github.com/poppingmoon/aria/pull/315 - a fix to prevent pull to refresh from jumping to the top when reading from old to new and having pending new notes.

The behavior I now see is, that after opening Aria, it seems to load the post I last viewed. It does not load new notes, which is a problem, because when I then pull to refresh, instead of loading the next note and allowing me to read from old to new, it jumps to latest note, which brings me back to issue https://github.com/poppingmoon/aria/issues/305. But in order to load new posts I do have to pull to refresh.

This is related but not identical to what is requested in the issue here. Do you want a separate follow-up issue for the new behavior?

**Comment by poppingmoon** [comment]
Thanks for reporting the problem.
I tried this and noticed that the "Jump to (lastViewedDate)" banner at the top does not appear after #315.
I will fix this and implement the remembering position feature at the same time so you don't have to create a new issue.
